src/main/java/com/twilio/oai/api/ApiResources.java

```diff
@@ -12,22 +12,32 @@
 public class ApiResources {
     private String apiName;
     private String apiPath;
+    private String apiListPath;
+    private String apiContextPath;
```

Comment:
Thoughts about extending ApiResources for PhpApiResources for properties specific to Php library? Since the plan is for all libraries to use ApiResources. @charan678 we did discuss on this today.